test/sslapitest.c

```diff
@@ -1936,17 +2046,27 @@ static int test_tlsext_status_type(void)
      * Now actually do a handshake and check OCSP information is exchanged and
      * the callbacks get called
      */
-    SSL_CTX_set_tlsext_status_cb(cctx, ocsp_client_cb);
-    SSL_CTX_set_tlsext_status_arg(cctx, &cdummyarg);
-    SSL_CTX_set_tlsext_status_cb(sctx, ocsp_server_cb);
-    SSL_CTX_set_tlsext_status_arg(sctx, &cdummyarg);
+    if (multi_stapling) {
+        cdummyarg = 1;
+        SSL_CTX_set_tlsext_status_cb(cctx, ocsp_client_cb_multi);
+        SSL_CTX_set_tlsext_status_arg(cctx, &cdummyarg);
```

Comment:
`cdummyarg` is set to 1. 
I don't see its value changed ?
And `ocsp_client_cb_multi` is testing `if (*argi == 2)`

Reply:
I see that `cdummyarg` is assigned to 2 at line 2098.
Yet I don't see that it has any effect, e.g., via. `SSL_CTX_set_tlsext_status_arg(cctx, &cdummyarg);`

Reply:
The address is stored in `ctx->ext.status_arg` and when the callback function is called, the address to the variable dummyarg is provied. 
If it has changed meanwhile the new value is evaluated in the callback function. It's working and it was not introduced by me.